app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="recipe-<%= @recipe.id %>">
+<h1>Recipe Info Page</h1>
+  <p>Name: <%= @recipe.name %></p>
+  <p>Complexity: <%= @recipe.complexity %></p>
+  <p>Genre: <%= @recipe.genre %></p>
+  <br>
+  <h2>Ingredients:</h2>
+  <ul>
+  <% @ingredients.each do |ingredient| %>
+    <li><%= ingredient.name %></li>
+  </ul> 
+    <% end %>
+  <p> Total Cost of All Ingredients: $ <%= @total_cost %></p>
```

Comment:
Since this `@total_cost` variable is coming frmo the controller, you're not actually using your `total_cost` method on the Recipe model. Be careful if this was a refactor that when you move those pieces around, they all line up again at the end.